common/src/clue/scala/queries/common/UserPreferencesQueriesGQL.scala

```diff
@@ -125,28 +128,61 @@ object UserPreferencesQueriesGQL {
           saturation
           brightness
         }
+      }
+    """
+  }
+
+  case class LucumaUserPreferences(
+    val aladinMouseScroll:  Option[Boolean],
```

Comment:
Why the `val`s?

Reply:
Also, do you think it's possible to decode directly into `GlobalPreferences`? I guess it would take a custom `Decoder`.

Reply:
Could be, though in that case I'd change the db to make the columns no nullable, that's probably the best